rfcs/0148-taskcluster-worker-location.md

```diff
@@ -0,0 +1,48 @@
+# RFC 148 - TASKCLUSTER_WORKER_LOCATION
+* Comments: [#148](https://api.github.com/repos/taskcluster/taskcluster-rfcs/issues/148)
+* Proposed by: @djmitche
+
+# Summary
+
+This proposal defines an environment variable, `TASKCLUSTER_WORKER_LOCATION`, containing this information in an easily-digested format.
+
+## Motivation
+
+Workers often need information about where they are running, for example to select nearby mirrors or other resources.
+
+Taskcluster does not currently provide this in a documented, supported fashion.
+
+# Details
+
+This proposes to introduce `$TASKCLUSTER_WORKER_LOCATION` as an environment variable available within tasks run by docker-worker and generic-worker.
+
+The content has the format `<cloud>/<details>`.
```

Comment:
I would be inclined to make this json, rather than a format that will need to be parsed by consumers.

Reply:
JSON still needs to be parsed! Many of the places Firefox CI currently uses `TASKCLUSTER_WORKER_GROUP` are in shell scripts, where parsing JSON is rather tricky.  What do you see as the benefit of JSON?

Reply:
I am generally not a fan of json in single-env-var for single-use like this, as it makes it harder to reason about and harder to parse. (more overhead) but I could be convinced.

Reply:
We could just take the easy way out and  have `TASCKLUSTER_WORKER_LOCATION` and `TASKCLUSTER_WORKER_LOCATION_JSON`?